contrib/oci-seccomp-gen/parse_arguments.go

```diff
@@ -0,0 +1,71 @@
+package ociseccompgen
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	types "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// parseArguments takes a list of arguments (delimArgs)  and a pointer to a
+// corresponding syscall struct. It parses and fills out the argument information
+func parseArguments(delimArgs []string) (*[]types.Arg, error) {
```

Comment:
This comment does not seem to match the function signature (I don't see any input `*types.Syscall`).